biz.aQute.bndlib/bnd.bnd

```diff
@@ -8,7 +8,6 @@ Bundle-Description: bndlib: A Swiss Army Knife for OSGi
 	aQute.lib.spring
 
 Export-Package: \
-	aQute.bnd.annotation.*;-noimport:=true,\
```

Comment:
bndlib has long contained these packages. I don't think it is wise to remove them now. Perhaps on 5.0.

Reply:
It's still in there! This is just to avoid double exportation warning. The packages are still contained and exported. This is a side effect of the new `package-info.java` usage in biz.aQute.bnd.annotation`